dietza/whats-cookin
# Begin building Recipe class and test files

# Pull Request Template:

#### What's this PR do?
Adds the Recipe class in Recipe.js, adds and passes corresponding tests in recipe-test.js

#### Where should the reviewer start?
Start in the Recipe.js class file.
Consult project spec / rubric for adherence:
Overview the dependencies and sample data sets at the top of the recipe-test.js testing file.

#### How should this be manually tested?
The reviewer should manually test this by using the previously installed mocha and chai testing frameworks to run `npm test`.
From a local copy of the repository, run `npm test ../test/Recipe-test.js`

#### Any background context you want to provide?
The reviewer should be aware that the datasets provided (sampleIngredients, sampleUsers, and sampleRecipes) on lines 7 - 632 of the recipe-test.js testing suite are shallow copies of the full datasets available in respective files within the data directory.
